pokemon-minisite/src/app/components/my-map/my-map.component.scss

```diff
@@ -0,0 +1,55 @@
+@use '../../style/variables' as *;
+@use '../../style/mixin' as *;
+
+span{
+    display: flex;
+    justify-content: space-between;
+    margin: 20px;
+}
+
+.buttons-container{
+    display: flex;
+    flex-direction: row;
```

Comment:
You don't need this. its the default direction

Reply:
I need it ):